app/renderer/components/tabs/content/favIcon.js

```diff
@@ -143,9 +146,22 @@ const styles = StyleSheet.create({
     animationIterationCount: 'infinite'
   },
 
-  icon__default: {
+  icon__loading_colorLight: {
+    filter: filter.makeWhite
+  },
+
+  icon__defaultIcon: {
     WebkitMaskRepeat: 'no-repeat',
     WebkitMaskPosition: 'center',
-    WebkitMaskImage: `url(${defaultIconSvg})`
+    WebkitMaskImage: `url(${defaultIconSvg})`,
+    WebkitMaskSize: '12px',
+    backgroundColor: color.mediumGray
+  },
+
+  icon__defaultIcon_reducedSize: {
+    WebkitMaskSize: '10px'
+  },
+  icon__defaultIcon_colorLight: {
```

Comment:
would you please add a blank line above this? https://github.com/brave/browser-laptop/blob/master/docs/style.md#always-add-space-between-styles-objects

Reply:
thanks - done